README.md

```diff
@@ -26,10 +26,10 @@ Before installation, please make sure you have already installed the following t
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project `git clone https://github.com/{your-username}/good-first-issue-finder.git`
```

Comment:
```suggestion
2. Clone the project
    ```bash
      git clone https://github.com/YOUR_USERNAME/good-first-issue-finder.git
    ```
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Unreadable, Efficient, Inelegant, Maintainable

Reply:
Thanks @krshkodes for your suggestions.
I will make the necessary changes.